README.md

```diff
@@ -48,6 +48,12 @@ arguments. For example:
 WebServiceClient client = new WebServiceClient.Builder(6, "ABCD567890").build();
```

Comment:
Hmm, line 44 says it can take an optional options array, but that looks incorrect to me. If you agree, should we remove that part of the sentence?

Reply:
Yes, I agree. https://github.com/maxmind/minfraud-api-java/blob/f90cb5255f6224d1e60033af69bc367f8304a891/src/main/java/com/maxmind/minfraud/WebServiceClient.java#L113-L116